apps/meteor/client/views/omnichannel/directory/contacts/hooks/useCurrentContacts.ts

```diff
@@ -0,0 +1,15 @@
+import type { GETLivechatRoomsParams, OperationResult } from '@rocket.chat/rest-typings';
+import { useEndpoint } from '@rocket.chat/ui-contexts';
+import type { UseQueryResult } from '@tanstack/react-query';
+import { useQuery } from '@tanstack/react-query';
+
+export const useCurrentContacts = (
+	query: GETLivechatRoomsParams,
+): UseQueryResult<OperationResult<'GET', '/v1/livechat/visitors.search'>> => {
+	const currentContacts = useEndpoint('GET', '/v1/livechat/visitors.search');
+
+	return useQuery(['current-contacts', query], () => currentContacts(query), {
+		// TODO: Update this to use an stream of room changes instead of polling
+		cacheTime: 0,
+	});
```

Comment:
I don't think I follow the `TODO` and cache configuration. If this is not intentional I recommend removing.

Reply:
Yes you are right, I have removed cacheTime key  from the query as you suggested